### DISCUSSION TITLE
chore: 🔧 Use "Code Connect" to show Code-Ressources in Figma

### DISCUSSION THREAD

**Discussion by LeaLeRose** [synergy-design-system/synergy-design-system#524] [RFCs]
# Description

with new dev mode updates from Figma, Code Connect, it is possible to integrate our dev resources into the dev mode.

> Code Connect is the developer bridge from your component codebase to Figma. With Code Connect, bring your design system component code directly into Figma's [Dev Mode](https://help.figma.com/hc/en-us/articles/15023202277399-Use-code-snippets-in-Dev-Mode#01H8CZ5SQRYZE7A36EW1KBWGS1). Preview example components that mirror the framework of your production code.

https://www.figma.com/developers/code-connect

![Image](https://github.com/synergy-design-system/synergy-design-system/assets/142008971/9ab3ce96-3895-48d1-8a90-53bb2ed87f18)

![Image](https://github.com/synergy-design-system/synergy-design-system/assets/142008971/a329e606-7a70-45b2-8001-e50d39bd3293)

# Rationale

**Comment by schilchSICKAG** [comment]
Done :)
